multimodal/src/autogluon/multimodal/predictor.py

```diff
@@ -1780,10 +1786,11 @@ def evaluate(
                 metric_data=metric_data,
                 metric_name=self._eval_metric_name.lower(),
             )
+            score = {k.lower(): v for k, v in score.items()}
             if metrics_is_none:
                 results = score
             else:
-                results.update({per_metric: score[per_metric] for per_metric in metrics})
+                results.update({per_metric: score[per_metric.lower()] for per_metric in metrics})
```

Comment:
the comput_score will output all scores. If we do not do this, users will have to provide every entity group as metric, which will be troublesome if there are a lot of entity groups (saying, 10, 20, 100, etc.)

Reply:
If users provide invalid metric names, we still compute all the possible scores. Then it would encounter error when collecting the results. Could we validate the provided metrics before doing the computation?

Reply:
Sure, it will throw an key error even if we don't validate it.